### PR TITLE
Default hubs and slots restrictions for real life

### PR DESCRIPTION
**Pull request by Tantrix**

### CODE REVIEW

**Review by blasti — APPROVED** [review]

File: examples/cfg/cfg.tbl [hunk 1]
```diff
@@ -201,12 +201,12 @@ return {
...
     min_slots = {  -- min slots (array of integer)
 
-        [ 0 ] = 2,  -- unreg (default = 2 slots)
-        [ 10 ] = 2,  -- guest (default = 2 slots)
-        [ 20 ] = 2,  -- reg (default = 2 slots)
-        [ 30 ] = 2,  -- vip (default = 2 slots)
-        [ 40 ] = 2,  -- svip (default = 2 slots)
-        [ 50 ] = 2,  -- server (default = 2 slots)
+        [ 0 ] = 1,  -- unreg (default = 1 slot)
+        [ 10 ] = 1,  -- guest (default = 2 slot)
```
Comment (Sopor) [inline examples/cfg/cfg.tbl:205]: This is not correct `[ 10 ] = 1,  -- guest (default = 2 slot)`

**Comment by Sopor** [comment]
I don't understand why increase it? 100 hubs and slots are totally ridiculous.

**Comment by Tantrix** [comment]
> 
> 
> I don't understand why increase it? 100 hubs and slots are totally ridiculous.

Well...not.

изображение

**Comment by Sopor** [comment]
Maybe that is real for you but it is not real for me. If you accept so many hubs maybe you can change it instead? Now all the rest of us will have to decrease it to a more acceptable level. 20 was perfect as it was.

**Comment by Tantrix** [comment]
It is not about you and me, it's about real-world practice. Deal with it any way you want.

**Comment by Sopor** [comment]
I don't know how you do it in russia but i have not seen any hub that allow so many hubs and it is not between you and me... This is only you...

**Comment by Tantrix** [comment]
> 
> 
> i have not seen any hub that allow so many hubs ... This is only you...

ORLY? Let me show you then.

изображение

Most **and** biggest hubs have no hubs\slots limitations at all. Period.

File: examples/cfg/cfg.tbl (hunk 1 above)
Comment (Tantrix) [reply examples/cfg/cfg.tbl:205]: Yepp(

**Comment by Sopor** [comment]
You seems to allow 250 hubs (if that is your hub) and even if it will be changed to 100 you have to change it to 250 (or whatever you allow). So i see no reason to change it to 100, you have to edit it whatever it is as default.

**Comment by Kungen123** [comment]
are you for real now ?

**Comment by Kungen123** [comment]
Blasti you think this is ok ?

**Comment by blasti** [comment]
> Blasti you think this is ok ?

Yes, because:

1.  Long term luadch users configured their hub long ago. Correct me if I'm wrong, but I suppose you guys just use your old config, if upgrading your hub to a new release?
2. If you configure your hub for the first time, you have to adjust the settings anyway. Which is not such a hard task, using "Find and Replace" in your $EDITOR.
3. I personally would really appreciate it being able to log into more then 20 hubs the same time. So these new settings suit me well.

**Pull request merged**